components/raftstore/src/coprocessor/dispatcher.rs

```diff
@@ -693,6 +694,19 @@ mod tests {
             ctx.bypass = self.bypass.load(Ordering::SeqCst);
             false
         }
+
+        fn post_exec_admin(
+            &self,
+            ctx: &mut ObserverContext<'_>,
+            _: &Cmd,
+            _: &RaftApplyState,
+            _: &RegionState,
+            _: &mut ApplyCtxInfo<'_>,
+        ) -> bool {
+            self.called.fetch_add(18, Ordering::SeqCst);
```

Comment:
In this test, we assign unique index for each observer functions so that we can test  all of the observer functions we want to be called are called. All numbers below 18 are used for other functions, so we use 18 for post_exec.